ui/app/routes/vault/cluster/secrets/backend/configuration/index.js

```diff
@@ -66,10 +66,12 @@ export default class SecretsBackendConfigurationRoute extends Route {
     switch (type) {
```

Comment:
It might be useful above this method to mention that `id` here _is_ the path where the backend is mounted (and often referred to just as `backend`) since there's only one config per mount. I know for future devs switching between `backend` and `id` can be confusing verbiage (it still confuses me!)